blogReader.py

```diff
@@ -15,6 +16,7 @@ def __init__(self) -> None:
         self.__new = []
         self.__getArticles()
         self.__cacheArticlesID()
+        self.handler = logging.getLogger(__name__)
```

Comment:
`logging.getLogger` returns a `Logger` object the name `self.handler` is a bit confusing!

Reply:
Name updated to make it clearer !

Reply:
tks !

Reply:
[6b64cd9](https://github.com/iScsc/iscsc.fr-blog-notify/pull/8/commits/6b64cd98d063d0c48613b1eeb4555ee1cac525af)